packages/ftp/package.json

```diff
@@ -21,17 +21,18 @@
   "homepage": "https://github.com/zazuko/barnard59",
   "dependencies": {
     "ftp": "^0.3.10",
-    "readable-stream": "^3.6.0",
+    "path-scurry": "^1.10.1",
```

Comment:
What's that? @ludovicm67

Reply:
Without this, I'm getting the following error while running the tests:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'path-scurry' imported from /***/barnard59/packages/ftp/node_modules/glob/dist/esm/glob.js
    at packageResolve (node:internal/modules/esm/resolve:844:9)
    at moduleResolve (node:internal/modules/esm/resolve:901:20)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at nextResolve (node:internal/modules/esm/hooks:865:28)
    at resolve (file:///***/barnard59/node_modules/tsm/loader.mjs:1:627)
    at nextResolve (node:internal/modules/esm/hooks:865:28)
    at Hooks.resolve (node:internal/modules/esm/hooks:303:30)
    at handleMessage (node:internal/modules/esm/worker:196:24)
    at Immediate.checkForMessages (node:internal/modules/esm/worker:138:28)
    at process.processImmediate (node:internal/timers:478:21)
```

Reply:
Strange

Reply:
If you remove that change, then the `node_modules` directory and the lock file, and then reinstall everything and if you run `npm run test` for the ftp package, you should be able to reproduce the issue.

Including it was then working well for me. The thing is that I'm not sure if it's only impacting the tests or the whole package, that's why I added it as a dependency and not a dev dependency.

Reply:
Actually, I could not reproduce this

Reply:
Even by doing the following from the root:

```sh
rm -rf node_modules package-lock.json
npm i
cd packages/ftp
npm run test
```

?

Reply:
Yes. Cannot reproduce in node 18 nor in node 20